components/StyledDropdown.js

```diff
@@ -112,6 +113,18 @@ export const Dropdown = styled(({ children, trigger, ...props }) => {
           }
         }
       }}
+      onKeyDown={e => {
```

Comment:
There's a regression that I think comes from this handler; it is not possible anymore to ope on a link in a dropdown using `Enter`:

![Peek 2021-04-02 10-21](https://user-images.githubusercontent.com/1556356/113397477-68a93400-939d-11eb-9dc1-648ad243d92c.gif)

Reply:
@Betree : For this one it's my mistake to add the `onKeyDown` listener. We already seem to have a `onFocus` and even when the dropdown is focused it auto opens it. Therefore this `onKeyDown` is not needed I think. I've removed it and added a eslint exception to this case instead. 😄